functions/getWidgetContent.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const getGitHubWidgetContent = require('./lib/get-github-widget-content');
+
+const widgetHandlerMap = {
+	github: getGitHubWidgetContent
+};
+
+const getWidgetContent = async ({ context, req }) => {
+  const {
+    query: { widget }
+  } = req;
+
+  if (!widgetHandlerMap[widget]) {
+	return {
```

Comment:
I'm not sure I agree with the response structure being set in this file. This module should return the data or throw an error, and the calling handler can decide how to handle success or failure.